files/01_setup_disabling_of_inactive_freeipa_users.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Set up this instance to disable inactive FreeIPA users.
+function setup {
+  # Disable the "KDC:Disable Last Success" password plugin feature.
+  # We don't want this feature enabled because we want to be able to
+  # disable inactive users, which requires us to be able to determine
+  # the last time a user authenticated.
+  password_plugin_features=$(ipa config-show \
+    |
+    # --quiet means no printing unless the p command is used
+    sed --quiet "s/^\s*Password plugin features:\s*\(.*\)/\1/p" \
```

Comment:
Is there a reason it is not like this?

```suggestion
    # --quiet means no printing unless the p command is used
    | sed --quiet "s/^\s*Password plugin features:\s*\(.*\)/\1/p" \
```